public/guides/COMMIT_MESSAGE_GUIDE.md

```diff
@@ -0,0 +1,80 @@
+# Effective Commit Messaging
+
+Clear and informative commit messages contribute significantly to a project's success. This guide
+outlines the importance of well-structured commit messages and descriptions.
+
+## Basic commit message structure
+
+A well-crafted commit message is structured to convey concise and actionable information about the
+changes made. It typically consists of a single-line summary and an optional extended description.
+
+### Summary line
+
+A good summary line is:
+
+- **Directive**: start with a concise imperative verb to indicate the action,
+  e.g.: `add`, `fix`, `update`.
+- **Specific**: it answers the question: What was done in this commit?
```

Comment:
To align with all the other lists in this page (or change every other items to lowercase):
```suggestion
- **Directive**: Start with a concise imperative verb to indicate the action,
  e.g.: `add`, `fix`, `update`.
- **Specific**: It answers the question: What was done in this commit?
```